ts/features/pushNotifications/analytics/__tests__/index.test.ts

```diff
@@ -236,6 +246,136 @@ describe("pushNotifications analytics", () => {
       new_notification_status: notificationPermissionsEnabled
     });
   });
+  [MESSAGES_ROUTES.MESSAGES_HOME, ROUTES.SETTINGS_MAIN].forEach(route => {
+    it(`'trackPushNotificationsBannerVisualized' should have expected event name and properties for '${route}'`, () => {
+      const mockMixpanelTrack = getMockMixpanelTrack();
+
+      void trackPushNotificationsBannerVisualized(route);
+
+      expect(mockMixpanelTrack.mock.calls.length).toBe(1);
+      expect(mockMixpanelTrack.mock.calls[0].length).toBe(2);
+      expect(mockMixpanelTrack.mock.calls[0][0]).toBe("BANNER");
+      expect(mockMixpanelTrack.mock.calls[0][1]).toEqual({
+        event_category: "UX",
+        event_type: "screen_view",
+        banner_id: "push_notif_activation",
+        banner_page:
+          route === "MESSAGES_HOME" ? "MESSAGES_HOME" : "SETTINGS_MAIN",
+        banner_landing: "os_notification_settings"
+      });
+    });
+    it(`'trackPushNotificationsBannerTap' should have expected event name and properties for '${route}'`, () => {
+      const mockMixpanelTrack = getMockMixpanelTrack();
+
+      void trackPushNotificationsBannerTap(route);
+
+      expect(mockMixpanelTrack.mock.calls.length).toBe(1);
+      expect(mockMixpanelTrack.mock.calls[0].length).toBe(2);
+      expect(mockMixpanelTrack.mock.calls[0][0]).toBe("TAP_BANNER");
+      expect(mockMixpanelTrack.mock.calls[0][1]).toEqual({
+        event_category: "UX",
+        event_type: "action",
+        banner_id: "push_notif_activation",
+        banner_page:
+          route === "MESSAGES_HOME" ? "MESSAGES_HOME" : "SETTINGS_MAIN",
```

Comment:
```suggestion
          route,
```
since this is a literal type, why not directly use it?

Reply:
Because it is a tracked data that appears in Mixpanel reports. Let's say that we just do `banner_page: route`. The test succeeds. Mixpanel boards are created, with this value in the reports (MESSAGES_HOME and SETTINGS_MAIN).
In the future, someone changes the value of the literal. Test still succeed, no warning at all but Mixpanel boards start receiving data that was not mapped (and nobody may be aware of it for a while).

With this redundancy in the test, if someone changes the literal value, the test fails so we have an error that should trigger the author (of the change) to reflect upon the impact on mixpanel data.

Reply:
Done on 4478970